easybuild/easyconfigs/n/nanocompore/nanocompore-1.0.0rc3-2-intel-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,65 @@
+easyblock = 'PythonBundle'
+
+name = 'nanocompore'
+version = '1.0.0rc3-2'
+versionsuffix = "-Python-%(pyver)s"
+
+homepage = 'https://nanocompore.rna.rocks/'
+description = "Nanocompore identifies differences in ONT nanopore sequencing raw signal corresponding to RNA modifications by comparing 2 samples"
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+    ('tqdm', '4.47.0'),
+    ('matplotlib', '3.2.1', versionsuffix),
+    ('Seaborn', '0.10.1', versionsuffix),
+    ('scikit-learn', '0.23.1', versionsuffix),
+    ('statsmodels', '0.11.1', versionsuffix),
+    ('poetry', '1.0.9', versionsuffix),
+    ('PyYAML', '5.3'),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+# poetry has unnecessarily restrictive dependencies
+local_nanocompore_preinstallopts = """sed -i'' 's/\^/>=/g' pyproject.toml && """
+local_nanocompore_preinstallopts += """sed -i'' 's/~/>=/g' pyproject.toml && """
+
+
+local_bedparse_preinstallopts = """sed -i'' 's/argparse//' setup.py && """
```

Comment:
Here should be a blank line

Reply:
Maybe a comment why removing argparse - eg. that it is part of python stdlib since 3.6+?

Reply:
changed, commented